templates/supplier_app/company_creation.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block styles %}
+{% load static %}
+{% endblock %}
```

Comment:
If you don't add any style then you do not need to add this block

Reply:
Its correct I am working on that template on next ticket